package.json

```diff
@@ -23,9 +23,9 @@
   },
   "devDependencies": {
     "grunt": "^1.0.3",
-    "grunt-contrib-clean": "^1.1.0",
+    "grunt-contrib-clean": "^2.0.0",
```

Comment:
The deps update shouldn't be in this commit. They are different changes.

Reply:
So do we actually want to update these?

If so, will make a separate PR for this 👌